### ISSUE TITLE
Repository with scripts and services detected with both "main" in has structure - face_classification

### ISSUE THREAD

**Issue by rosafilgueira** [SoftwareUnderstanding/code_inspector#239]
The repository [face classification](https://github.com/oarriaga/face_classification) has a collection of scripts and a web service, but looking into the repo. The code_inspector extract all this information. See bellow:

<img width="1070" alt="Screen Shot 2021-08-06 at 11 18 18" src="https://user-images.githubusercontent.com/6940078/128496184-b0fd02e7-3e26-4839-9963-41cbfb45697c.png">

For our future heuristic, in which we will implement the type classification taking into "has_strucure" (for script and services), this will be an interesting case - if we have services and scripts, both with main ..... something to think about.

**Comment by rosafilgueira** [comment]
I have updated the code, so it also checks the mention of the file, not only taking into account the fileNameBase, as well, as the extension (e.g. faces.py). 

```
mentioned_in_readme": elem["file"]["fileNameBase"]+"."+elem["file"]["extension"]  in readme
```

<img width="1187" alt="Screen Shot 2021-08-12 at 14 33 15" src="https://user-images.githubusercontent.com/6940078/129205939-880ca17f-aa5a-44a1-9251-39cb15a42a9b.png">

Note that for the scripts with main, it already the implementation had into account the complete name of the file (fileNameBase + Extension) - so I havent modified the code there. 

But I did had to modify the code also for "script without main ... with body"
